src/index.ts

```diff
@@ -33,3 +33,4 @@ export {
     buildSignedCredentialForExistingAccount,
 } from './credentialDeploymentTransactions';
 export { isAlias, getAlias } from './alias';
+export { deserializeContractState } from './deserialization';
```

Comment:
The file could also be called something with schema, to clarify what is being deserialized

Reply:
When it is used externally it is all one module, so it's mostly for internal structure, and I think it's fine like this until we get too much de-serialization, and we would need to split it up.